kubevirtci, presubmit: Introduce k8s-1.25-ipv6

oshoval opened the pull request:
Depends on https://github.com/kubevirt/kubevirtci/pull/912

oshoval commented:
/rehearse

oshoval commented:
```
error: resource mapping not found for name: "calico-kube-controllers" namespace: "kube-system" from "/provision/cni.yaml": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
```
Need to bump calico or so

oshoval commented:
/rehearse

oshoval commented:
/rehearse

dhiller commented on github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:
```diff
@@ -529,6 +529,34 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
```
Let's start with `always_run: false` and `optional: true`.

oshoval replied:
np, it was still draft ;]

oshoval commented:
Addressed comments,
Removed the manual apply of https://github.com/kubevirt/kubevirtci/pull/912

/hold
until https://github.com/kubevirt/kubevirtci/pull/912 is merged

dhiller reviewed (approved):
/approve

Since the new lane is triggered manually, you could also merge it right away, but I'll leave that up to you :)

oshoval commented:
Right thanks
lets merge it

/hold cancel
